_data/team-academic.yml

```
@@ -39,16 +39,18 @@
 - name: "Yair"
   surname: "Grinberger"
   title: "Hebrew University of Jerusalem"
-  thumbnailUrl: "generic.jpg"
+  thumbnailUrl: "Yair_Grinberger.jpg"
```

Please keep image name consistent: `Firstname Lastname.extension`
```suggestion
  thumbnailUrl: "Yair Grinberger.jpg"
```